src/test/java/jeryl/fyp/storage/JsonAdaptedStudentTest.java

```diff
@@ -93,15 +94,15 @@ public void toModelType_nullEmail_throwsIllegalValueException() {
     public void toModelType_invalidProjectName_throwsIllegalValueException() {
         JsonAdaptedStudent student =
                 new JsonAdaptedStudent(VALID_NAME, VALID_STUDENT_ID, VALID_EMAIL, INVALID_PROJECT_NAME,
-                        VALID_TAGS);
+                        VALID_PROJECT_STATUS, VALID_TAGS);
         String expectedMessage = ProjectName.MESSAGE_CONSTRAINTS;
         assertThrows(IllegalValueException.class, expectedMessage, student::toModelType);
     }
 
     @Test
     public void toModelType_nullProjectName_throwsIllegalValueException() {
         JsonAdaptedStudent student = new JsonAdaptedStudent(VALID_NAME, VALID_STUDENT_ID, VALID_EMAIL,
-                null, VALID_TAGS);
+                null, VALID_PROJECT_STATUS, VALID_TAGS);
         String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, ProjectName.class.getSimpleName());
         assertThrows(IllegalValueException.class, expectedMessage, student::toModelType);
     }
```

Comment:
Other possible tests to be added here: for invalid project status but similar format to that of the other fields. Notice how the ones for projectName are similar to the other fields, got null, invalid, and so on.